frontend/src/components/Cart.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import CartItem from './CartItem';
+
+const Cart = () => {
+    const [cartItems, setCartItems] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
probs simpler to remove this, no network calls means this is basically always instant unless we fuck up